docs/source/en/gpu_selection.md

```diff
@@ -55,40 +55,75 @@ deepspeed --num_gpus 2 trainer-program.py ...
+Only GPUs 0 and 2 are "visible" to PyTorch and are mapped to `cuda:0` and `cuda:1` respectively.  
+To reverse the order (use GPU 2 as `cuda:0` and GPU 0 as `cuda:1`):
+
 
 ```bash
 CUDA_VISIBLE_DEVICES=2,0 torchrun trainer-program.py ...
 ```
 
-You can also set the `CUDA_VISIBLE_DEVICES` environment variable to an empty value to create an environment without GPUs.
+To run without any GPUs:
 
 ```bash
 CUDA_VISIBLE_DEVICES= python trainer-program.py ...
 ```
 
-> [!WARNING]
-> As with any environment variable, they can be exported instead of being added to the command line. However, this is not recommended because it can be confusing if you forget how the environment variable was set up and you end up using the wrong GPUs. Instead, it is common practice to set the environment variable for a specific training run on the same command line.
+You can also control the order in which CUDA devices are enumerated using `CUDA_DEVICE_ORDER`:
+
+- Order by PCIe bus ID (matches `nvidia-smi`):
+
+    ```bash
+    export CUDA_DEVICE_ORDER=PCI_BUS_ID
+    ```
 
-`CUDA_DEVICE_ORDER` is an alternative environment variable you can use to control how the GPUs are ordered. You can order according to the following.
+- Order by compute capability (fastest first):
+
+    ```bash
+    export CUDA_DEVICE_ORDER=FASTEST_FIRST
+    ```
+
+</hfoption>
+<hfoption id="Intel">
+
+```bash
+ZE_AFFINITY_MASK=0,2 torchrun trainer-program.py ...
+```
 
-1. PCIe bus IDs that matches the order of [`nvidia-smi`](https://developer.nvidia.com/nvidia-system-management-interface) and [`rocm-smi`](https://rocm.docs.amd.com/projects/rocm_smi_lib/en/latest/.doxygen/docBin/html/index.html) for NVIDIA and AMD GPUs respectively.
+Only GPUs 0 and 2 are "visible" to PyTorch and are mapped to `xpu:0` and `xpu:1` respectively.  
+To reverse the order (use GPU 2 as `xpu:0` and GPU 0 as `xpu:1`):
 
 ```bash
-export CUDA_DEVICE_ORDER=PCI_BUS_ID
+ZE_AFFINITY_MASK=2,0 torchrun trainer-program.py ...
 ```
 
-2. GPU compute ability.
+To run without any Intel GPUs:
 
 ```bash
-export CUDA_DEVICE_ORDER=FASTEST_FIRST
+ZE_AFFINITY_MASK= python trainer-program.py ...
 ```
 
-The `CUDA_DEVICE_ORDER` is especially useful if your training setup consists of an older and newer GPU, where the older GPU appears first, but you cannot physically swap the cards to make the newer GPU appear first. In this case, set `CUDA_DEVICE_ORDER=FASTEST_FIRST` to always use the newer and faster GPU first (`nvidia-smi` or `rocm-smi` still reports the GPUs in their PCIe order). Or you could also set `export CUDA_VISIBLE_DEVICES=1,0`.
\ No newline at end of file
+You can also control the enumeration order of Intel GPUs by PCIe bus ID using:
+
+```bash
+export ZE_ENABLE_PCI_ID_DEVICE_ORDER=1
+```
+
```

Comment:
add such like: You can go [here](https://github.com/oneapi-src/level-zero/blob/master/README.md?plain=1#L87) for more details of `ZE_ENABLE_PCI_ID_DEVICE_ORDER`